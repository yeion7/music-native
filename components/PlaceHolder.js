/* @flow */

import React, { Component } from "react";
import { View, Text, Dimensions } from "react-native";

import styled from "styled-components/native";

var { height } = Dimensions.get("window");

const Wrapper = styled.View`
  flex: 1;
  justify-content: center;
  align-items: center;
  height: ${height - height * 0.2}

`;

const Emoji = styled.Text`
  font-size: 50;
`;

const PlaceHolder = () => (
  <Wrapper>
    <Emoji>🎧</Emoji>
    <Text>Busca tu canción favorita</Text>
  </Wrapper>
);

export default PlaceHolder;
