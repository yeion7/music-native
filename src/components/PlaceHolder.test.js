import React from "react";
import "react-native";

import PlaceHolder from "./PlaceHolder";

import renderer from "react-test-renderer";

describe("renders test ", () => {
  it("render PlaceHolder component", () => {
    const tree = renderer.create(<PlaceHolder />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
