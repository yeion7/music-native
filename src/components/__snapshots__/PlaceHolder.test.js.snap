// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders test  render PlaceHolder component 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "height": 533.6,
        "justifyContent": "center",
      },
      undefined,
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontSize": 50,
        },
        undefined,
      ]
    }
  >
    ðŸŽ§
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Busca tu canciÃ³n favorita
  </Text>
</View>
`;
